TEMPLATE-SPEC.md

```diff
@@ -9,69 +9,123 @@ Requires: spec-xyz | ip-xyz
 
 # Spec-<Number>: <Title>
 
-## Summary
+## Overview
```

Comment:
Let's call it Summary or Abstract - this is a standard for the documents.

Reply:
Let's keep Overview. Summary is short and part of the overview. Abstract is the opposite of a spec, it's something lacking definition.

Reply:
* NEP templates requires "Summary": https://github.com/near/NEPs/blob/master/nep-0000-template.md
* W3C is also using Summary
* Latest EIPs are using "Abstract": https://github.com/ethereum/EIPs/blob/master/EIPS/eip-6900.md
* Algorand is using "Abstract": https://github.com/algorandfoundation/ARCs/blob/main/ARC-template.md

TBH, I don't recall any design process which uses "overview"